.github/workflows/triage_update_status.yml

```diff
@@ -0,0 +1,97 @@
+name: 'Set Issue Status'
+on:
+  # makes this workflow reusable
```

Comment:
It can be reused for the `triage_closed_issue_comment` action but I believe a small refactor is needed in that action for it to actually use it.

Reply:
It can be reused to set the status of an issue to whatever you want.  Current use case is having any issues marked as stale to be added to the Awaiting Response column (since we are waiting for the end user to show activity).  In the future we could use this action to move PR's to the Contributor PR In Progress column, or even issues that get marked as feature being moved to the Feature Review column.